vcs/vcs2vtk.py

```diff
@@ -29,7 +29,7 @@
     "eqdc",
     "tmerc",
     "stere",
-    "lcca",
```

Comment:
@durack1 These are internal names, that probably should not be used by users. Not sure if it works if you use the internal name. Users should use the names from https://github.com/UV-CDAT/vcs/blob/master/vcs/projection.py